docs/content/preview/explore/ysql-language-features/pg-extensions.md

```diff
@@ -190,6 +191,40 @@ SELECT levenshtein('Yugabyte', 'yugabyte'), metaphone('yugabyte', 8);
 (1 row)
 ```
 
+### passwordcheck example
+
+To enable the passwordcheck extension, add `passwordcheck` to `shared_preload_libraries` in the PostgreSQL server configuration parameters using the YB-TServer [`--ysql_pg_conf_csv`](../../../reference/configuration/yb-tserver/#ysql-pg-conf-csv) flag:
+
+```sh
+--ysql_pg_conf_csv="shared_preload_libraries=passwordcheck"
+```
+
+When enabled, if a password is considered too weak, it's rejected with an error. For example:
+
+```sql
+yugabyte=# create role test_role password '1234';
```

Comment:
May be use the nicer examples from this file.

https://github.com/yugabyte/yugabyte-db/blob/master/src/postgres/contrib/passwordcheck/sql/passwordcheck.sql